Anlab.Mvc/Views/Shared/_OrderDetails.cshtml

```diff
@@ -147,135 +147,143 @@
     }
 </div>
 
-@if (!string.IsNullOrWhiteSpace(Model.OrderDetails.LabComments))
-{
-    <div class="form-group">
-        <strong>Lab Comments</strong>
-        <div class="well well-sm" style="white-space: pre-wrap">@Model.OrderDetails.LabComments</div>
-    </div>
-}
+    @if (!Model.HideLabDetails)
+    {
+        <div class="form-group">
+            <strong>Paid <i class="analysisTooltip fa fa-info-circle" title="This is not an indicator that the final money movement has happened."></i></strong>

```

Comment:
@jSylvestre I would much rather this help tip tell them what the status means than what it doesn't mean